src/Matej.php

```diff
@@ -2,19 +2,61 @@
 
 namespace Lmc\Matej;
 
+use Http\Client\HttpClient;
+use Http\Message\MessageFactory;
+use Lmc\Matej\Http\RequestManager;
+use Lmc\Matej\Http\ResponseDecoderInterface;
+use Lmc\Matej\RequestBuilder\RequestBuilderFactory;
+
 class Matej
 {
     public const CLIENT_ID = 'php-client';
     public const VERSION = '0.0.0';
 
     /** @var string */
-    private $clientId;
+    private $accountId;
     /** @var string */
     private $apiKey;
+    /** @var RequestManager */
+    protected $requestManager;
```

Comment:
Why `protected`?

We already have `protected` getter so subclass have acces to `$requestManager`.

Reply:
True, fixed.